### ISSUE TITLE
Always show play button on videos that can't preload (i.e. YouTube)

### ISSUE THREAD

**Issue by colejd** [ccrama/Slide-iOS#887]

**Comment by ccrama** [comment]
We don't show YouTube inline, should we? Otherwise the play button would just open the player view

**Comment by colejd** [comment]
I don't think we should. It's too unstable. Let's just have the play button open the player view.

**Comment by colejd** [comment]
It doesn't even need to be a button. It just needs to be an image since the behavior when the video is tapped is to open the player anyway.
